resources/views/inputs/tile-selection.blade.php

```diff
@@ -23,24 +29,27 @@ class="space-y-4 {{ $class }}"
             if (this.allSelected) {
                 checkAllValue = false;
             }
-
             for (const optionKey in this.options) {
                 this.options[optionKey].checked = checkAllValue;
             }
         }
     }"
 >
-    <div class="space-y-6">
+    <div class="{{ $wrapperClass }}">
         <div class="flex flex-col space-y-4 md:space-y-0 md:flex-row md:justify-between {{ $description ? 'md:items-end' : 'md:items-center' }}">
-            <div class="flex flex-col">
-                <div class="text-lg font-bold text-theme-secondary-900">
-                    {{ $title }}
-                </div>
+            @unless($withoutTitle && ! $description)
```

Comment:
Is it not possible to just check if `$title` is passed in as a property, the same way we do the description?